supporting_content/dimensions/resourceid.md

```diff
@@ -0,0 +1,55 @@
+
+## Examples
+| Provider  | Scenario                                          | Pattern                                                                                          |
+|-----------|---------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| AWS       | ARN provided which includes a resource name or id | arn:partition:service:region:account-id:resource-id <br>arn:partition:service:region:account-id:resource-type/resource-id<br> arn:partition:service:region:account-id:resource-type:resource-id|
+| AWS       | Some cases, just a resource-id is provided        | I-0b21f4c4434558933 OR my-cur-bucket                                                             |
+| AWS       | Tax, API call charges etc.                        | (null)                                                                                           |
+| GCP       | Resource based                                    | //container.googleapis.com/projects/<project_id>/locations/<location>/clusters/<cluster>         |
+| GCP       | Non-resource based (e.g. network egress)          | (null)                                                                                           |
+| Microsoft | Resource group resources                          | /subscriptions/<guid>/resourceGroups/<name>/providers/<provider>/<resource-type>/<resource-name> |
+| Microsoft | Subscription resources                            | /subscriptions/<guid>/providers/<provider>/<resource-type>/<resource-name>                       |
+| Microsoft | Tenant resources                                  | /providers/<provider>/<resource-type>/<resource-name>                                            |
+| Microsoft | Purchase charges                                  | (null)                                                                                           |
+| Microsoft | Reservation purchases                             | /providers/Microsoft.Capacity/reservationOrders/<guid>                                           |
+| Microsoft | Savings plan purchases                            | /providers/Microsoft.Capacity/savingsPlanOrders/<guid>                                           |
+
+### Documentation
+
+AWS:
+> <https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html>
+
+Interesting note on the "The resource identifier" which is at the end of the ARN: This is the name of the resource, the ID of the resource, or a [resource path](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#arns-paths). Some resource identifiers include a parent resource (sub-resource-type/parent-resource/sub-resource) or a qualifier such as a version (resource-type:resource-name:qualifier)."
+
+Microsoft:
+
+GCP:
+> <https://cloud.google.com/apis/design/resource_names> <https://cloud.google.com/asset-inventory/docs/resource-name-format> 
+
+## Discussion / Scratch space
```

Comment:
nit: Should we move action items to issues and potential discussion points that don't have a specific action to discussions? Not sure if we want to commit these to the repo, where they won't be tracked/closed.

Reply:
My goal was to provide a sneak peek for the things we discussed / considered as a part of building this. It's not really to track anything. Agree on those needing to be issues in the repo.

Reply:
Cleaned up more of the discussion items

Reply:
Hmm... if it's minutes, we're missing a bunch. I thought this was the open items we didn't close, which we should be tracking somewhere... 🤔

Reply:
It's not minutes. This is just things we discussed and/or considered while building this. Minutes aren't a part of the PR. These are so people can get a sense for what was discussed and considered as a part of building the spec content.

If anything requires decisions beyond this work or needs to be tracked, they should be opened as an issue in the repo with the discussion topic label.